src/Symfony/Component/Notifier/Bridge/JoliNotif/JoliNotifOptions.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\JoliNotif;
+
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Ahmed Ghanem <ahmedghanem7361@gmail.com>
+ */
+final class JoliNotifOptions implements MessageOptionsInterface
+{
+    public function __construct(private ?string $iconPath = null, private array $extraOptions = [])
```

Comment:
For consistency with the code base:

```suggestion
    public function __construct(
        private ?string $iconPath = null,
        private array $extraOptions = [],
    ) {
```